src/main/java/org/reaktivity/k3po/nukleus/ext/internal/behavior/NukleusPartition.java

```diff
@@ -64,7 +64,7 @@
     NukleusPartition(
         Path partitionPath,
         StreamsLayout layout,
-        LongFunction<NukleusServerChannel> lookupRoute,
+        BiFunction<Long, Long, NukleusServerChannel> lookupRoute,
```

Comment:
Define a `LongLongFunction<R>` functional interface with method `R apply(long value1, long value2)` to avoid unnecessary auto-boxing of `long` to `Long` during lookup.

Reply:
Done